docs/contributing/05-gh-actions.md

```diff
@@ -0,0 +1,206 @@
+    - [AbsaOSS/k3d-action](#absaossk3d-action)
+    - [zgosalvez/github-actions-ensure-sha-pinned-actions](#zgosalvezgithub-actions-ensure-sha-pinned-actions)
+    - [ahmadnassri/action-workflow-queue](#ahmadnassriaction-workflow-queue)
+    - [autotelic/action-wait-for-status-check](#autotelicaction-wait-for-status-check)
+    - [gaurav-nelson/github-action-markdown-link-check](#gaurav-nelsongithub-action-markdown-link-check)
+    - [slackapi/slack-github-action](#slackapislack-github-action)
+  - [Not allowed GH Actions](#not-allowed-gh-actions)
+    - [shmokmt/actions-setup-github-comment](#shmokmtactions-setup-github-comment)
+    - [shmokmt/actions-setup-tfcmt](#shmokmtactions-setup-tfcmt)
+    - [CondeNast/conventional-pull-request-action](#condenastconventional-pull-request-action)
+
+## Introduction
+
+Github Actions are a powerful tool to automate almost every task of the development cycle. The ecosystem of Github Actions is growing very fast and one can find a prepared Action for a lot of activities. But like all other usages of Open Source Software, Github Actions have to be handled carefully because an action is running external code on our code in our Github repositories.
+
+## Restrictions of use
+
+The usage of external Github Actions is restricted centrally on Github.com organization level. Only Actions in the kyma-project org or approved external Actions are allowed. Actions created by Github or [verified creators](https://github.com/marketplace?type=actions&verification=verified_creator) are also allowed. The list of approved Github Actions can be found in the [Appendix A: Approved Github Actions](#appendix-a-approved-github-actions). Approved Actions are added by using the hash sign of the version commit to ensure that really the approved version of the Action is used.
+
+![Github Actions policy](./assets/gh-actions-policies.png)
+
+Github Actions have read and write permissions on repositories but they are not allowed to create and merge pull requests.
+
+![Github Actions permissions](./assets/gh-actions-peermissions.png)
+
+## Github Actions Security Best Practices
+
+Even the usage of Github Actions is restricted there are some threats left. The responsibility to target this threats is on the developer using Github Actions. GitGuardian ([blog article on how to secure Github Actions](https://blog.gitguardian.com/github-actions-security-cheat-sheet/)) as well as Github itself ([Security hardening for Github Actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions)) are providing information on how to securely use Github Actions. Developers using Github Actions should make themselves aware about the security best practices in order to keep our code secure.
+
+**Github Actions Security Best Practices - Cheat Sheet from GitGuardian**
+![Github Actions Security Best Practices - Cheat Sheet](assets/GitHub-Actions-Security-Best-Practices_cheatsheet-1.png)
```

Comment:
There's no such file in the assets folder.

Reply:
I added a commit containing the file